cockatrice/src/window_main.cpp

```diff
@@ -455,7 +455,7 @@ void MainWindow::registerError(Response::ResponseCode r, QString reasonStr, quin
             QMessageBox::critical(this, tr("Registration denied"), tr("The email address provider used during registration has been blacklisted for use on this server."));
             break;
         case Response::RespTooManyRequests:
-            QMessageBox::critical(this, tr("Registration denied"), tr("Too many registration attempts, please try again later or contact the server operator for further details."));
+            QMessageBox::critical(this, tr("Registration denied"), tr("It appears you are attempting to register a new account on this server yet you already have an account registered and the server restricts the number of accounts a user can register.  Please contact the server operator for further assistance or to obtain your credential information."));
```

Comment:
> yet you already have an account registered 

Does that mean account with the same email address? If so, say that, maybe:

> ... but you already have an account registered to that email address. If you are trying to activate that account, <do x>. If you need to reset your password <do y>. Otherwise, contact the server operator for further assistance.